python-sdk/rebuff/detect_pi_vectorbase.py

```diff
@@ -1,21 +1,34 @@
-from typing import Dict
+from typing import Dict, List, Tuple, Union, Optional
 
 import pinecone
 from langchain.vectorstores.pinecone import Pinecone
+from langchain_community.vectorstores import Chroma
+from langchain_core.documents.base import Document
 from langchain_openai import OpenAIEmbeddings
 
+try:
+    import chromadb
+
+    chromadb_installed = True
+except ImportError:
+    print(
+        "To use Chromadb, please install rebuff with rebuff extras. 'pip install \"rebuff[chromadb]\"'"
+    )
+    chromadb_installed = False
+
 
-# https://api.python.langchain.com/en/latest/vectorstores/langchain.vectorstores.pinecone.Pinecone.html
 def detect_pi_using_vector_database(
-    input: str, similarity_threshold: float, vector_store: Pinecone
+    input: str,
+    similarity_threshold: float,
+    vector_store: Union[Pinecone, Optional[Chroma]],
```

Comment:
```suggestion
    vector_store: VectorStore,
```
And then you'd need to import `VectorStore` from langchain at the top of the file.